.github/workflows/ci.yml

```diff
@@ -0,0 +1,71 @@
+name: CI
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+
+jobs:
+  format:
+    name: Check Format
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 8
+        uses: actions/setup-java@v2
+        with:
+          java-version: '8.0'
+          distribution: 'adopt'
+      - run: |
+          ./dev/javafmt
+          if [[ $(git diff) != "" ]]
+          then
+              echo "code format error, please run the following commands:"
+              echo "   ./dev/javafmt"
+              exit 1
+          fi
+
+  integration-test:
+    name: Integration Test - ${{ matrix.tikv_version }}
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        tikv_version: [nightly, v5.0.4, v5.3.0, v5.4.0]
```

Comment:
oh, how about [v5.0.0, v5.1.0, v5.2.0, v5.3.0, v5.4.0]?

Reply:
I'm afraid it's too much for pull request ci. It's better to do it on release or nightly. Anyway, `tiup list tikv | awk '{print $1}'` should do the job.